lib/merchant.rb

```diff
@@ -1,5 +1,6 @@
 require 'bigdecimal'
 require 'bigdecimal/util'
+require 'date'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.